### ISSUE TITLE
AnsibleUndefinedVariable: 'ansible.vars.hostvars.HostVarsVars object' has no attribute 'ansible_default_ipv4'

### ISSUE THREAD

**Issue by rponline** [udima-university/ansible-jitsi-meet#10]
Tried to install all on one host following the USAGE.md  
on a typical cloud VM with Debian 9 
having DNS A and AAAA record configured.

The playbook got stuck here:

```
TASK [jitsi-meet : Nginx virtualhost config] *****************************************************************************************************
fatal: [meet.domainblabla.org]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'ansible.vars.hostvars.HostVarsVars object' has no attribute 'ansible_default_ipv4'"}

```
I guess it's about this line:
https://github.com/udima-university/ansible-jitsi-meet/blob/65b2a9472e4046000b2f09a584f9e01a86659e1c/roles/jitsi-meet/templates/nginx.conf#L93

My IP Config on the VM:
```
root@jitsimeet:~# ip route
default via 172.31.1.1 dev eth0 
172.31.1.1 dev eth0 scope link
```
I tried to add gather_facts = yes on different places. But it didn't help.  
I got around the error by just hardcoding the IP address.

**Comment by inittab** [comment]
Hi, what were the contents of your "hosts" file? Seems like you had a host listed in the [videobridges] section other than the one you were installing into.

**Comment by inittab** [comment]
Closing this as it was fixed in #11. Thanks, Alberto
